core/src/main/java/nl/nn/adapterframework/pipes/XmlFileElementIteratorPipe.java

```diff
@@ -196,8 +195,8 @@ public TimeOutException getTimeOutException() {
 	protected void iterateOverInput(Message input, PipeLineSession session, Map<String,Object> threadContext, ItemCallback callback) throws SenderException, TimeOutException {
 		InputStream xmlInput;
 		try {
-			xmlInput = new FileInputStream((String) input.asObject());
-		} catch (FileNotFoundException e) {
+			xmlInput = new FileInputStream(input.asString());
```

Comment:
Het zou mijn voorkeur hebben om `new FileInputStream(input.asString());`  te vervangen door `input.asInputStream()`.
Dat doet niet hetzelfde, maar is meer wat je zou (moeten) verwachten

Reply:
> Het zou mijn voorkeur hebben om `new FileInputStream(input.asString());` te vervangen door `input.asInputStream()`.
> Dat doet niet hetzelfde, maar is meer wat je zou (moeten) verwachten

Gezien ons gesprek op teams, toch maar laten zoals het is?

Reply:
ja inderdaad